draft-scenarios/clash/obelisk.tex

```diff
@@ -0,0 +1,55 @@
+% !TeX spellcheck = en_US
+\addscenariosection{1}{Clash Scenario}{The Obelisk}{\images/obelisk.png}
+
+\begin{multicols*}{2}
+
+\textbf{Author:} Pinkerton, biigrick, LAAMAKALA
+
+\textbf{Source:} \href{<https://discord.com/channels/740870068178649108/1232319328049954826>}{Archon discord}
+
+\textit{The map is torn, the path unclear — but the obelisks call. Rally your armies, master your magic, and stake your claim before another hero takes the prize.}
+
+\subsection*{\MakeUppercase{Scenario Length}}
+This Scenario plays out over 8-10 Rounds.
+
+\subsection*{\MakeUppercase{Player Setup}}
+\textbf{Player Count:} 2-6 FFA or Alliance
+
+Choose starting \textbf{Resources, Income, Units} and \textbf{Town Buildings} from any Scenario.
+
+\subsection*{\MakeUppercase{Map Setup}}
+Take the following Map Tiles and arrange them as shown in the Scenario map layout ($P$ stands for the number of players):
+
+\begin{itemize}
+  \item P × Starting (I) Map Tiles
+  \item 2P × Far (II-III) Map Tiles (Map Pool)
```

Comment:
Please see how map tile pool is defined in another scenario. An example: https://github.com/qwrtln/Homm3BG-mission-book/blob/main/coops/sentinels.tex#L35